instrumentation/aws_sdk/lib/opentelemetry/instrumentation/aws_sdk/messaging_helper.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Instrumentation
+    module AwsSdk
+      # MessagingHelper class provides methods for calculating messaging span attributes
+      class MessagingHelper
+        class << self
+          def queue_name(context)
+            topic_arn = context.params[:topic_arn]
+            target_arn = context.params[:target_arn]
+            phone_number = context.params[:phone_number]
+            queue_url = context.params[:queue_url]
```

Comment:
I'd consider moving these two down to immediately before the corresponding `return ... if ...`.

Reply:
Thanks, moved it. https://github.com/open-telemetry/opentelemetry-ruby/pull/1026/commits/5f34bc044a18fcfbab6ada684559511b8d76bd6b